syllabiShare/views.py

```diff
@@ -176,18 +186,33 @@ def schooladmin(request,domain=None):
     return redirect('/')
 
 
+def saved(request):
+    (template, context) = authenticate(request.user)
+    if template:
+        return render(request, template, context)
+
+    if 'unsave' in request.POST:
+        request.user.profile.saved.remove(Submission.objects.get(pk=request.POST['pk']))
+
+    found = request.user.profile.saved.filter(hidden=False)
+
+    return render(request, 'display.html', {'posts':found.order_by('dept','number'), 'AWS_S3_CUSTOM_DOMAIN':settings.AWS_S3_CUSTOM_DOMAIN,'saved': True, 'school':request.user.profile.school.name})
```

Comment:
Would it be easier if this was split up into its own template?

Reply:
I find it easy to use one template just because it will always look the same but I guess we could do something like have display-base.html then search.html, browse.html, and saved.html